includes/wikia/models/DesignSystemFooterModel.class.php

```diff
@@ -0,0 +1,429 @@
+						'key' => 'wikia-community-link-fan-contributor-program'
+					],
+					'href' => 'http://fandom.wikia.com/fan-contributor'
+				],
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-community-link-wam-score'
+					],
+					'href' => 'http://www.wikia.com/WAM'
+				],
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-community-link-help'
+					],
+					'href' => 'http://community.wikia.com/wiki/Help:Contents'
+				]
+			]
+		],
+		'create_wiki' => [
+			'description' => [
+				'type' => 'translatable-text',
+				'key' => 'wikia-create-wiki-description'
+			],
+			'links' => [
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-create-wiki-link-start-wikia'
+					],
+					'href' => 'http://www.wikia.com/Special:CreateNewWiki'
+				]
+			]
+		],
+		'community_apps' => [
+			'header' => [
+				'type' => 'line-text',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'wikia-community-apps-header'
+				]
+			],
+			'description' => [
+				'type' => 'translatable-text',
+				'key' => 'wikia-community-apps-description'
+			],
+			'links' => [
+				[
+					'type' => 'link-image',
+					'image' => 'company/store-appstore',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-community-apps-link-app-store'
+					],
+					'href' => 'https://itunes.apple.com/pl/developer/wikia-inc./id422467077'
+				],
+				[
+					'type' => 'link-image',
+					'image' => 'company/store-googleplay',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-community-apps-link-google-play'
+					],
+					'href' => 'https://play.google.com/store/apps/developer?id=Wikia,+Inc.'
+				]
+			]
+		],
+		'advertise' => [
+			'header' => [
+				'type' => 'line-text',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'wikia-advertise-header'
+				]
+			],
+			'links' => [
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-advertise-link-media-kit'
+					],
+					'href' => 'http://www.wikia.com/mediakit'
+				],
+				[
+					'type' => 'link-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'wikia-advertise-link-contact'
+					],
+					'href' => 'http://www.wikia.com/mediakit#contacts'
+				]
+			]
+		],
+	];
+
+	private $allowedVerticals = [
+		'games',
+		'movies',
+		'tv',
+	];
+
+	private $wikiId;
+
+	public function __construct( $wikiId ) {
+		parent::__construct();
+
+		$this->wikiId = $wikiId;
+	}
+
+	public function getData() {
+		$this->setLicensingAndVertical();
+
+		return $this->data;
+	}
+
+	private function setLicensingAndVertical() {
+		$params = [
+			'license' => $this->getLicenseData(),
+		];
+		$verticalData = $this->getVerticalData();
+
+		if ( !empty( $verticalData ) ) {
```

Comment:
Ideally we should still be returning the license, rather than nothing on the ~30,000 wikis this affects. I'd prefer the client to handle no vertical, but we can try to come up with a default which would also solve this.